TypeFox/open-collaboration-tools
# Add official VS Code extension

First version of the open collaboratio vscode extension. 

Contains the ability to Share Text editor state and the hosts file system

**How to test**
- build the extension with vsce package --no-dependencies
- install via vsix into Vscode
- start the open-collaboration-server (easiest is through the debug configuration)
- open multiple windows and open a workspace in the host window
- on the status bar you can find a OCT item. Clicking it should reveal the option to create or join a room
